CONTRIBUTING.md

```diff
@@ -69,24 +73,30 @@ For developers interested in expanding `pymatgen` for their own purposes, we rec
 Given that `pymatgen` is intended to be a long-term code base, we adopt very strict quality control and coding guidelines for all contributions to `pymatgen`. The following must be satisfied for your contributions to be accepted into `pymatgen`.
 
 1. **Unit tests** are required for all new modules and methods. The only way to minimize code regression is to ensure that all code is well-tested. Untested contributions will not be accepted.
-   To run the testsuite in you repository follow these steps
+   To run the testsuite in you repository follow these steps:
 
    ```sh
    cd path/to/repo
-   pip install -e .  # install the package in your environment as "editable" == dev package
-   PMG_TEST_FILES_DIR=$(pwd)/tests/files pytest tests  # run the test suite providing the path for the datafiles
```

Comment:
I believe setting `PMG_TEST_FILES_DIR` is not needed for editable mode, however I might still leave the "non-editable install + set PMG_TEST_FILES_DIR" as an option here, as there might be users who want to install the package in non-editable mode but still run tests, say [people who don't intend to develop `pymatgen` but just run tests to verify `pymatgen` integration](https://github.com/materialsproject/pymatgen/issues/4172).